rococo-parachains/tests/purge_chain_works.rs

```diff
@@ -15,50 +15,63 @@
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
 use assert_cmd::cargo::cargo_bin;
-use std::{convert::TryInto, fs, path::PathBuf, process::Command, thread, time::Duration};
+use std::{convert::TryInto, process::Command, thread, time::Duration};
 
 mod common;
 
 #[test]
 #[cfg(unix)]
 fn purge_chain_works() {
-	use nix::{
-		sys::signal::{kill, Signal::SIGINT},
-		unistd::Pid,
-	};
+	fn run_node_and_stop() -> tempfile::TempDir {
+		use nix::{
+			sys::signal::{kill, Signal::SIGINT},
+			unistd::Pid,
+		};
 
-	let base_path = "purge_chain_test";
+		let base_path = tempfile::tempdir().unwrap();
```

Comment:
@cecton I'm curious why you did this after you already gave us the `--tmp` flag in https://github.com/paritytech/substrate/pull/6221

Reply:
Oh, I get it. So you can check the directory later on.